AvatarIconViewer.plugin.js

```diff
@@ -150,49 +151,45 @@ var AvatarIconViewer = (() => {
 						copyLabel = "Copy Avatar Link";
 					}
 
-					if (_this.props.guild) {
+					if (_this.props.guild && _this.props.guild.icon) {
 						url = _this.props.guild.getIconURL().split("?")[0] + "?size=2048";
 						
-						if (_this.props.guild.getIconURL().includes("/a_")) {
+						if (url.includes("/a_")) {
 							url = url.replace(".webp", ".gif");
 						}
 						
 						viewLabel = "View Icon";
 						copyLabel = "Copy Icon Link";
 					}
 
-					if (_this.props.target.className.includes("emoji")) {
+					if (_this.props.target && typeof _this.props.target.className === 'string' && _this.props.target.className.includes("emoji")) {

```

Comment:
_this.props.target can be undefined, as can _this.props.target.className, I saw 2 errors thrown about this in my console